components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/util/FrameworkUtils.java

```diff
@@ -1558,6 +1558,17 @@ public static void publishEventOnUserRegistrationSuccess(Map<String, String> cla
         publishEventOnUserRegistration(claims, tenantDomain, event);
     }
 
+    public static void publishEventOnUserRegistrationSuccess(Map<String, String> claims, String tenantDomain,
+                                                             String userStoreDomain) {
```

Comment:
Switch tenant domain to last

Reply:
Fixed eeb7efb8b0cae14d83fcbafbee64e72c59c52776